src/Files.Uwp/UserControls/MultitaskingControl/HorizontalMultitaskingControl.xaml

```diff
@@ -590,7 +590,7 @@
                         Header="{x:Bind Header, Mode=OneWay}"
                         IconSource="{x:Bind IconSource, Mode=OneWay}"
                         Style="{StaticResource Local.TabViewItem}"
-                        ToolTipService.ToolTip="{x:Null}" />
+                        ToolTipService.ToolTip="{Binding HoverDisplayText}" />
```

Comment:
Does x:Bind work here?

Reply:
Will try it tonight.

Reply:
Doesn't seems to work like so @yaichenbaum.
I'm not really experienced with xaml though, so if someone else wants to give it a shot, please be my guest!

Reply:
Resolved in another line of comment.